.github/workflows/ci.yml

```diff
@@ -154,14 +154,21 @@ jobs:
       - name: Run tests
         env:
           SYMFONY_PHPUNIT_VERSION: ${{ matrix.phpunit_version }}
-          SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.strategy == 'lowest' && '9999' || '1' }}
+          OVERRIDE_SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.strategy == 'lowest' && 'max[total]=9999' || '' }}
         run: |
           ok=0
+          if [ -n "$OVERRIDE_SYMFONY_DEPRECATIONS_HELPER" ]; then
+            export SYMFONY_DEPRECATIONS_HELPER=$OVERRIDE_SYMFONY_DEPRECATIONS_HELPER
+          fi
```

Comment:
This avoid defining the `SYMFONY_DEPRECATIONS_HELPER` env variable and let the value defined in `phpunit.xml.dist`

Reply:
Oh. This is what I was missing before. Thank you!